scripts/asm-installer/install_asm

```diff
@@ -2563,6 +2594,11 @@ scrape_managed_urls() {
 }
 
 install_managed_components() {
+  if [[ "${USE_MCP_CNI}" -eq 1 ]]; then
+    info "Installing CNI into the cluster..."
```

Comment:
Whats our philosophy on overwriting user installed (oss?) components? should we warn/prompt/fail or anything?

Reply:
So far the customer feedback has been: if they have to add an extra flag to do something (e.g. `--cni-managed`) then they're aware that dependencies are going to be installed and might overwrite things. Soon(tm) when we have more defined interactive and non-interactive modes, we'll have a more consistent stance.